src/style.css

```
@@ -0,0 +1,132 @@
+body,
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+  display: inline-block;
+}
+
+.card {
+  width: 188px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-left: 16px;
+}
+
+.card__iMac {
+  margin-top: 32px;
+  margin-left: 5px;
+  width: 160px;
+}
+
+.card__iMacMark {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-top: 40px;
+  width: 180px;
+}
+
+.iMacMarkText {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.iMacMarkCode {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-top: 4px;
+  color: #616070;
+}
+
+.stars {
+  display: flex;
+  margin-top: 16px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--rate_1 .stars__star:nth-child(1),
+.stars--rate_2 .stars__star:nth-child(-n + 2),
+.stars--rate_3 .stars__star:nth-child(-n + 3),
+.stars--rate_4 .stars__star:nth-child(-n + 4),
+.stars--rate_5 .stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.reviews {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  padding-left: 13px;
+}
+
+.card__priceBlock {
+  margin-top: 24px;
+}
+
+.priceBlockText {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+  padding-right: 51px;
+}
+
+.priceBlockprice {
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__buyBlock {
+  display: flex;
+  justify-content: center;
+  padding-top: 16px;
+}
+
+.buyBlockLink {
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+
+  text-transform: uppercase;
+  text-decoration: none;
+
+  background-color: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  padding: 12px 55px;
+  margin-left: -20px;
+  margin-bottom: 16px;
+}
+
+.buyBlockLink:hover {
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

You can add default border with transparent color and change this color on hover

```suggestion
  margin-bottom: 16px;
  border: 1px solid transparent;
}

.buyBlockLink:hover {
  background-color: #fff;
  border-color: #00acdc;
```